build.gradle

```diff
@@ -346,11 +346,11 @@ dependencies {
     implementation 'org.checkerframework:checker-qual:3.29.0'
     implementation "com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}"
     implementation "com.fasterxml.jackson.core:jackson-databind:${jacksonDataBindVersion}"
+    implementation "org.opensearch:performance-analyzer-commons:1.0.0-SNAPSHOT"
```

Comment:
`1.0.0-SNAPSHOT` is meant for tracking active development and we use the release version otherwise. 

Let us fix this in follow-up PR.